### PR TITLE
Feature/go pro/phase 2/part 3/pull request

### PR DESCRIPTION
**Pull request by sultanmyrza**
┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201016280880500/1201323068735515) by [Unito](https://www.unito.io)
┆Created By: Tammy Yang

### CODE REVIEW

File: src/app/features/settings/go-pro/go-pro.page.ts [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+    public toastController: ToastController,
+    private goProMediaService: GoProMediaService,
+    private router: Router,
+    private goProBluetoothService: GoProBluetoothService
+  ) {}
+  ngOnDestroy(): void {}
+
+  ngOnInit() {
+    this.restoreBluetoothConnection();
+    // this.scanForBluetoothDevices();
```
Comment (shc261392) [inline src/app/features/settings/go-pro/go-pro.page.ts:69]: Unused code

File: src/app/features/settings/go-pro/go-pro-media-item-detail-on-camera/go-pro-media-item-detail-on-camera.component.ts [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+    // }
+
+    // if (newtorkStatus.connectionType == 'cellular') {
+    //   const allowed = await this.allowUploadWithMobileInternet();
+    //   if (allowed) {
+    //     this.startUploadToCapture();
+    //     return;
+    //   }
+    // }
+  }
```
Comment (shc261392) [inline src/app/features/settings/go-pro/go-pro-media-item-detail-on-camera/go-pro-media-item-detail-on-camera.component.ts:102]: Unused code

File: src/app/features/settings/go-pro/go-pro.page.ts [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+    // @ts-ignore
+    const { wifiSSID, wifiPASS } = await this.getGoProWiFiCreds();
+
+    try {
+      // this.presentToast(`Wifi.connectPrefix`);
+      // console.log(`Connecting to ${wifiSSID} with password ${wifiPASS}`);
+      // await Wifi.connectPrefix({
+      //   ssid: wifiSSID,
+      //   password: wifiPASS,
+      // })
```
Comment (shc261392) [inline src/app/features/settings/go-pro/go-pro.page.ts:206]: Unused code

**Review by shc261392 — CHANGES_REQUESTED** [review]

File: android/app/src/main/res/xml/network_security_config.xml [hunk 4]
```diff
@@ -2,5 +2,7 @@
 <network-security-config>
     <domain-config cleartextTrafficPermitted="true">
         <domain includeSubdomains="false">localhost</domain>
+        <domain includeSubdomains="true">10.5.5.9</domain>
+        <domain includeSubdomains="true">192.168.8.155</domain>
```
[the remark this answers is not in the document]
Comment (sultanmyrza) [reply android/app/src/main/res/xml/network_security_config.xml]: removed in this [commit](https://github.com/numbersprotocol/capture-lite/pull/1034/commits/d9517171fce12243bdaf3962a95cc2e2ba6ed40b) 👍

**Review by shc261392 — APPROVED** [review]

**Pull request merged**